src/core/public/mocks.ts

```diff
@@ -25,3 +34,35 @@ export { injectedMetadataServiceMock } from './injected_metadata/injected_metada
 export { legacyPlatformServiceMock } from './legacy/legacy_service.mock';
 export { notificationServiceMock } from './notifications/notifications_service.mock';
 export { uiSettingsServiceMock } from './ui_settings/ui_settings_service.mock';
+export { overlayServiceMock } from './overlays/overlay_service.mock';
+
+function createCoreSetupMock() {
+  const mock: jest.Mocked<CoreSetup> = {
+    http: httpServiceMock.createSetupContract(),
+    fatalErrors: fatalErrorsServiceMock.createSetupContract(),
+    notifications: notificationServiceMock.createSetupContract(),
+    uiSettings: uiSettingsServiceMock.createSetupContract(),
+  };
+
+  return mock;
+}
+
+function createCoreStartMock() {
+  const mock: jest.Mocked<CoreStart> = {
```

Comment:
`jest.Mocked` mocks only methods. Use
```typescript
const mock = { ...
```
this will fix the problem with https://github.com/elastic/kibana/pull/39351/files#diff-7ac462c1f389a60d4bb0b5f2e388fcd9R83

Reply:
I agree, it's better to let TS infer the types here. What I wanted to do is get a tslint error if the mock doesn't satisfy the `CoreStart` signature. The best I could come up with was https://github.com/elastic/kibana/pull/39351/files#diff-df31ff762b324f9b2c195223f6de7c89R49